packages/grafana-data/src/types/app.ts

```diff
@@ -7,6 +7,7 @@ export enum CoreApp {
   Dashboard = 'dashboard',
   Explore = 'explore',
   Unknown = 'unknown',
+  CloudAlerting = 'cloud-alerting',
```

Comment:
should there not be an Alerting option here as well?

Reply:
Not yet, but we will probably add it in PRs to come.